### PR TITLE
Add user guide features

### LINKED ISSUE #27: Update UG Features

### PR DESCRIPTION
**Pull request by Md-Fazil**
Closes #27.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -45,102 +45,93 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/Chocolate Milk`.
```
Comment (JayChenYJ) [inline docs/UserGuide.md]: Perhaps standadise it to ITEM_NAME.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -45,102 +45,93 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
+  e.g `n/NAME [e/EXPIRY_DATE]` can be used as `n/Olive Oil e/2020-10-10` or as `n/Olive Oil`.
```
Comment (JayChenYJ) [inline docs/UserGuide.md]: Same for here :)

File: docs/UserGuide.md [hunk 3]
```diff
@@ -45,102 +45,93 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/favourite`, `t/favourite t/drink` etc.
 
 * Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+  e.g. if the command specifies `n/NAME l/LOCATION`, `l/LOCATION n/NAME` is also acceptable.
```
Comment (JayChenYJ) [inline docs/UserGuide.md]: Same for here :)

File: docs/UserGuide.md [hunk 4]
```diff
@@ -45,102 +45,93 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
 * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+  e.g. if you specify `e/2020-10-10 e/2020-08-08`, only `e/2020-08-08` will be taken.
 
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
```
Comment (JayChenYJ) [inline docs/UserGuide.md]: Our list takes in an optional parameter, so may consider removing it from this line.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -45,102 +45,93 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* `add n/Chocolate Milk l/Kitchen Refrigerator q/2`
+* `add n/Sunshine Bread l/Bedroom q/10 e/2020-01-01 t/favourite`
 
-### Listing all persons : `list`
+### Listing all items : `list`
```
Comment (JayChenYJ) [inline docs/UserGuide.md:81]: our list should be able to in an optional field, location.

File: docs/UserGuide.md [hunk 6]
```diff
@@ -45,102 +45,93 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-Shows a list of all persons in the address book.
+Shows a list of all items in the inventory.
 
 Format: `list`
 
-### Editing a person : `edit`
+### Updating an item's details : `update`
 
-Edits an existing person in the address book.
+Edits an existing item in the inventory.
```
Comment (JayChenYJ) [inline docs/UserGuide.md]: Should consider changing edits to updates :)

File: docs/UserGuide.md [hunk 7]
```diff
@@ -45,102 +45,93 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
 Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+* `find Chocolate` returns `chocolate` and `Chocolate Milk`
+* `find potato chip` returns `Potato Biscuit`, `chocolate chip`<br>
+
 
 ### Deleting a person : `delete`
```
Comment (JayChenYJ) [inline docs/UserGuide.md]: Should be deleting an item instead :)

File: docs/UserGuide.md [hunk 8]
```diff
@@ -45,102 +45,93 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
-Deletes the specified person from the address book.
+Deletes the specified item from the inventory.
 
 Format: `delete INDEX`
```
Comment (JayChenYJ) [inline docs/UserGuide.md:128]: Check google docs for delete method for StoreMando :)

**Review by JayChenYJ — COMMENTED** [review]
WIP

File: docs/UserGuide.md (hunk 2 above)
Comment (Md-Fazil) [reply docs/UserGuide.md]: Fixed

File: docs/UserGuide.md (hunk 3 above)
Comment (Md-Fazil) [reply docs/UserGuide.md]: Fixed.

File: docs/UserGuide.md (hunk 4 above)
Comment (Md-Fazil) [reply docs/UserGuide.md]: Fixed.

File: docs/UserGuide.md (hunk 6 above)
Comment (Md-Fazil) [reply docs/UserGuide.md]: fixed.

File: docs/UserGuide.md (hunk 7 above)
Comment (Md-Fazil) [reply docs/UserGuide.md]: fixed.

**Review by JayChenYJ — APPROVED** [review]
r+

**Pull request merged**